README.md

```diff
@@ -168,6 +169,63 @@ Make sure that you have your Home Assistant Container network set to 'host', as
 
 - [Google Home timers card](https://github.com/DurgNomis-drol/google_home_timers_card) by [@DurgNomis-drol](https://github.com/DurgNomis-drol)
 
+## Troubleshooting
+
+### Collecting useful log data
+
+Here are the steps to generate useful log data:
+
+1. Temporary log level change.
+    1. Visit [![Open your Home Assistant instance and show your service developer tools.](https://my.home-assistant.io/badges/developer_services.svg)](https://my.home-assistant.io/redirect/developer_services/)
```

Comment:
Can you replace `my.home-assistant.io` badges with regular links please? It should be cleaner this way.

Reply:
Why? I think it would be very HA to use the new MY feature 😄

Reply:
Yeah, only badges. I think they're standing out too much.

Reply:
`MY` feature is actually pretty neat. Not sure if @KapJI has tried it before, but we already use it in `Integration Setup` on README

Reply:
@KapJI Yeah, they are quite big 😆 Lets hope they introduce some smaller ones in the future.

Reply:
> 
> 
> Can you replace `my.home-assistant.io` badges with regular links please? It should be cleaner this way.
I agree. I was trying to honor the way Home Assistant proposed it be used, but I don't like it. I will look briefly for smaller graphics. If not I will replace it with text.